drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -3223,7 +3223,7 @@ void RasterizerSceneGLES2::render_scene(const Transform &p_cam_transform, const
 
 		probe_interior = probe->probe_ptr->interior;
 
-	} else {
+	} else if (storage->frame.current_rt) {
```

Comment:
```
drivers/gles2/rasterizer_scene_gles2.cpp:3226:13: error: variable

      'viewport_width' is used uninitialized whenever 'if' condition is false

      [-Werror,-Wsometimes-uninitialized]

        } else if (storage->frame.current_rt) {

                   ^~~~~~~~~~~~~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3249:26: note: uninitialized use occurs

      here

        state.viewport_size.x = viewport_width;

                                ^~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3226:9: note: remove the 'if' if its

      condition is always true

        } else if (storage->frame.current_rt) {

               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3199:20: note: initialize the variable

      'viewport_width' to silence this warning

        int viewport_width, viewport_height;

                          ^

                           = 0

drivers/gles2/rasterizer_scene_gles2.cpp:3226:13: error: variable

      'viewport_height' is used uninitialized whenever 'if' condition is false

      [-Werror,-Wsometimes-uninitialized]

        } else if (storage->frame.current_rt) {

                   ^~~~~~~~~~~~~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3250:26: note: uninitialized use occurs

      here

        state.viewport_size.y = viewport_height;

                                ^~~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3226:9: note: remove the 'if' if its

      condition is always true

        } else if (storage->frame.current_rt) {

               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

drivers/gles2/rasterizer_scene_gles2.cpp:3199:37: note: initialize the variable

      'viewport_height' to silence this warning

        int viewport_width, viewport_height;

                                           ^

                                            = 0

2 errors generated.
```

I still don't see the point in adding these checks, as `storage->frame.current_rt` is already used in all the code before these checks. What was the warning?

Reply:
If you can remove these changes (and maybe open another PR / issue to discuss what the actual warning was), the rest seems good to merge.